WordPress/Classes/Utility/Media/ItemProviderMediaExporter.swift

```diff
@@ -67,15 +70,18 @@ final class ItemProviderMediaExporter: MediaExporter {
 
             // Retaining `self` on purpose.
             do {
-                let copyURL = try self.mediaFileManager.makeLocalMediaURL(withFilename: url.lastPathComponent, fileExtension: url.pathExtension)
-                try FileManager.default.copyItem(at: url, to: copyURL)
```

Comment:
It's a bit unsafe to remove this code because of how `loadFileRepresentation` works:

> This method writes a copy of the file’s data to a temporary file, which the system deletes when the completion handler returns.

Reply:
Good catch. I think it's best to keep the copying code. I have created another solution in https://github.com/wordpress-mobile/WordPress-iOS/pull/24595.